### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by SolomonAfutu2003**
In this branch, I
. Added a new section called "About Myself"
. Added new HTML and CSS files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+          <img src="img/Snapshoot Portfolio.png" alt="" />
+          <div class="Card1-P">
+            <h4>Tonic</h4>
+            <div class="Card1-tech">
+              <p class="P1">Canopy</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P2">Back End Dev</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P3">2015</p>
+            </div>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure that you use the `<p>` tag only for the paragraph. here you can use div or `<ul> `tag :+1: 
This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+          <img src="img/Snapshoot Portfolio (1).png" alt="" />
+          <div class="Card2-P">
+            <h4>Multi-Post Stories</h4>
+            <div class="Card2-tech">
+              <p class="P1">Canopy</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P2">Back End Dev</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P3">2015</p>
+            </div>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure that you use the `<p>` tag only for the paragraph. here you can use div or `<ul> `tag :+1: 
This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+        <div class="Card4">
+          <img src="img/Snapshoot Portfolio (3).png" alt="" />
+          <div class="Card4-P">
+            <h4>Multi-Post Stories</h4>
+            <div class="Card4-tech">
+              <p class="P1">Canopy</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P2">Back End Dev</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P3">2015</p>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure that you use the `<p>` tag only for the paragraph. here you can use div or `<ul> `tag :+1: 
This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+        <div class="Card3">
+          <img src="img/Snapshoot Portfolio (2).png" alt="" />
+          <div class="Card3-P">
+            <h4>Tonic</h4>
+            <div class="Card3-tech">
+              <p class="P1">Canopy</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P2">Back End Dev</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P3">2015</p>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure that you use the `<p>` tag only for the paragraph. here you can use div or `<ul> `tag :+1: 
This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+          </div>
+          <div class="Frame">
+            <p>Frameworks</p>
+            <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
+          </div>
+          <div class="skill">
+            <p>Skills</p>
+            <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
+          </div>
+        </div>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly make sure that you use the `<p>` tag only for the paragraph. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li`or `<ul> `tag :+1: 
This way will be semantically correct and the screen readers and other technologies will thank you.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+              <i class="fa fa-medium fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href=""
+              ><i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i
+            ></a>
+          </ul>
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="Language">
+          <h2>Languages</h2>
```
Comment (MaganaAsiati) [inline index.html]: - Kindly add the arrow button as shown in the Figma design. your design must stick to the Figma design as mutch as possible :+1: 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @SolomonTheCoder 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️

- Great job adding hover into your button but kindly follow the button styling: Active, Hover, and Pressed (see design). you must use the same color as is shown in the Figma design 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 7]
```diff
@@ -1 +1,175 @@
...
+
+- npm test
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
```
Comment (Kossi-stack) [inline README.md]: - [OPTIONAL] Kindly replace Author1 with your name.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button">See projects</button>
+        </div>
```
Comment (Kossi-stack) [inline index.html]: - [ ] As requested by the previous reviewer kindly fix the issue and let your code look like this
Screenshot 2023-03-23 at 16 05 41

File: index.html [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
```
Comment (Kossi-stack) [inline index.html]: -  [ ] Kindly follow the Figma Template (Ex: color, font size, etc...)

- Your Design
Screenshot 2023-03-23 at 16 00 00

- [ ] Figma  Design
Screenshot 2023-03-23 at 16 02 57

File: index.html [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+            <a href=""
+              ><i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i
+            ></a>
+          </ul>
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="Language">
+          <h2>
+            Languages <i class="fa fa-angle-down fa-2x" aria-hidden="true"></i>
+          </h2>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly fix the drop-down arrow

- [ ] Your design
Screenshot 2023-03-23 at 16 18 28

- [ ] Figma Design
Screenshot 2023-03-23 at 16 19 59

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @SolomonTheCoder,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

 - [OPTIONAL] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: Finish Mobile Version PR".

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Shedrack-Sunday — APPROVED** [review]
# STATUS: APPROVED ♻️♻️
Hi @SolomonTheCoder 

To highlight 👍🏽
- GitHub flow rightfully used ✅
- Professional README.md file ✅
-  No linter errors ✅

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**